extensions/wikia/Wall/js/Wall.js

```diff
@@ -19,7 +19,7 @@
 			this.deletedMessages = {};
 			this.isMonobook = window.skin && window.skin === 'monobook';
 			this.hasMiniEditor = typeof window.wgEnableMiniEditorExt !== 'undefined' && !this.isMonobook;
-			this.title = window.wgTitle.split('/')[0];
+			this.title = window.wgTitle;
```

Comment:
Didn't find place on MessageWall where you could get ```wgTitle``` that contains ```/```. That's why I think this is valid fix.
The only place in Wall/Forum where I was able to create page that ```wgTitle``` contains ```/``` was board and this ```.split()``` was causing issues there.